src/Tizen.NUI/src/public/Events/GestureDetector.cs

```diff
@@ -121,6 +121,32 @@ public View GetAttachedView(uint index)
             return ret;
         }
 
+        /// <summary>
+        /// Handles the event for a given view and touch input.
+        /// This method should only be called when SetGeometryHittestEnabled is set to true.
+        /// It processes the touch input and attempts to recognize gestures based on the provided view and touch data.
+        /// </summary>
+        /// <param name="view">The view associated with the gesture detector.</param>
+        /// <param name="touch">The touch input data to analyze for gestures.</param>
+        /// <returns>True if the event was handled successfully, otherwise false.</returns>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public bool HandleEvent(View view, Touch touch)
+        {
+            bool ret = Interop.GestureDetector.HandleEvent(SwigCPtr, View.getCPtr(view), Touch.getCPtr(touch));
+            if (NDalicPINVOKE.SWIGPendingException.Pending) throw NDalicPINVOKE.SWIGPendingException.Retrieve();
```

Comment:
```suggestion
            NDalicPINVOKE.ThrowExceptionIfExist();
```

Please use more shortten method.